o3po/includes/class-o3po-primary-publication-type.php

```diff
@@ -1475,6 +1482,7 @@ public function get_the_content( $content ) {
             $content .= '</header>';
             $content .= '<div class="entry-content">';
             $content .= '<p class="abstract">';
+            $content .= $settings->get_plugin_option('page_template_abstract_header');
```

Comment:
I think this should be wrapped in `esc_html()`, otherwise special characters of quotation marks in this settings field will yield invalid html.

Reply:
How should the abstract heading be implemented? I thought, that the heading level should be customisable, e.g. that the value for compositionally is `<h3>ABSTRACT</h3>` (that's why I preferred to use the template :) ) and be interpreted as HTML. Should the heading level be hard coded in the plugin?